package.json

```
@@ -0,0 +1,26 @@
+{
+  "name": "algorithmia-js",
+  "version": "1.0.0",
+  "description": "algorithmia-js client lib",
+  "main": "index.js",
+  "scripts": {
+    "test": "mocha-headless-chrome -f test/testRunner.html?key=\"$npm_config_key\" -r nyan "
```

I like how you made the key dynamic. My only suggestion would be to use an environment variable instead of reading from `npm_config_key`.
```suggestion
    "test": "mocha-headless-chrome -f test/testRunner.html?key=\"$API_KEY\" -r nyan"
```